Source/WebKit/UIProcess/Inspector/WebInspectorUIProxy.cpp

```diff
@@ -792,6 +793,19 @@ void WebInspectorUIProxy::setDeveloperPreferenceOverride(WebCore::InspectorBacke
         if (RefPtr inspectedPage = m_inspectedPage.get())
             inspectedPage->protectedPreferences()->setNeedsSiteSpecificQuirksInspectorOverride(overrideValue);
         return;
+
+    case InspectorBackendClient::DeveloperPreference::SiteIsolationDebugOverlay:
+        if (RefPtr inspectedPage = m_inspectedPage.get()) {
+            uint32_t siteIsolationRegion = (uint32_t)DebugOverlayRegions::SiteIsolationRegion;
```

Comment:
```suggestion
            auto siteIsolationRegion = static_cast<uint32_t>(DebugOverlayRegions::SiteIsolationRegion);
```

Reply:
actually is this even necessary?  could you just inline it instead?
```diff
 if (overrideValue && overrideValue.value())
-    debugOverlayRegions |= siteIsolationRegion;
+    debugOverlayRegions |= DebugOverlayRegions::SiteIsolationRegion;
 else
-    debugOverlayRegions &= ~siteIsolationRegion;
+    debugOverlayRegions &= ~DebugOverlayRegions::SiteIsolationRegion;
```

Reply:
The `static_cast` has to stay there otherwise it doesn't know which integral type to use.